e2e/for_with_range_and_if/main/main.neva

```diff
@@ -3,9 +3,11 @@ import { lists, fmt }
 const lst list<bool> = [true, false]
 
 def Main(start any) (stop any) {
-	ListToStream<bool>, For<bool>{PrintAsNum}, Wait
+	ListToStream<bool>, For<bool>{PrintAsNum}, Wait, Panic
```

Comment:
I've found the reason of deadlock - here `PrintAsNum` has outports `sig, err` while `For` expects `res, err`. So println sends to `:sig` but nobody reads from there. It's specific case of the #435 

Here's the correct implementation:

```neva
import { lists, fmt }

const lst list<bool> = [true, false]

def Main(start any) (stop any) {
	ListToStream<bool>, For<bool>{PrintAsNum}, Wait, Panic
	---
	:start -> $lst -> listToStream -> for
	for:res -> wait -> :stop
	for:err -> panic
}

def PrintAsNum(data bool) (res any, err error) {
	println fmt.Println?
	---
	(:data ? 1 : 0) -> println -> :res
}
```

Also no need for `t.Skip()` for this test